### DISCUSSION TITLE
Blazor Webassembly Debugging

### DISCUSSION THREAD

**Discussion by Postlagerkarte** [dotnet/aspnetcore#26024] [General] [answered]
A major reason why working with most MS technologies is a joy is the excellent tooling support. However, using VS 2019 there is a significant difference in debugging features between Blazor Server and Blazor WASM.
Is it fair to assume that the end goal is to provide Blazor WebAssembly with all the missing features and most importantly is there a roadmap for the missing debugger features somewhere?

**Comment by SteveSandersonMS** [comment, marked as answer]
Yes, it is a goal to fill in as many gaps as we reasonably can as time goes on. This is a big undertaking. We made a lot of progress in .NET 5 but there is still a long way to go.

cc @lewing in case there's any info about roadmap we can point to.

@Postlagerkarte If you have wishes about particular debugger features to prioritise, please consider posting an issue in the https://github.com/dotnet/runtime repo.
